### DISCUSSION TITLE
Tidak bisa upload gambar lewat c-k-editor

### DISCUSSION THREAD

**Discussion by lokersumatera** [mecha-cms/mecha#189] [Q&A]
Kang, setiap saya ingin upload gambar via c-k-editor, muncul error 444. itu kenapa ya kang?
![Screenshot 2022-02-10 155439](https://user-images.githubusercontent.com/48011166/153372126-b6fa42ab-1590-43b8-81e5-87b642c0e13e.png)

**Comment by taufik-nurrohman** [comment]
Mungkin dari pengaturan _server_?

https://www.google.com/search?q=error+status+444
